hypha/apply/users/views.py

```diff
@@ -159,7 +159,10 @@ class EmailChangeDoneView(TemplateView):
 
 @login_required()
 def become(request):
-    if not request.user.is_apply_staff:
+    if not settings.HIJACK_ENABLE:
+        raise Http404(_('Hijack feature is not enabled.'))
```

Comment:
Changed this to a 404 as well. Is it even needed now?

Reply:
Not needed. 

_Although we could have it for code readability purposes._